sentence-generator.js

```diff
@@ -0,0 +1,95 @@
+'a bad wife', 'a joke nobody will laugh about', 'sense of your life', 'a trip to Hong Kong', 'a trip to North Korea',
+'it', 'many mistakes', 'up for being so you', 'coffee for a friend']
+
+const caseTake = ['a vacation in Bahamas', 'a fall and hit your head', 'too much time playing this game', 
+`note of my maker's GitHub profile https://github.com/ribeiroAllison and check his awesome work`, 'two hours to get to work',
+'a wrong turn on your way home', 'a bottle and drink it down', 'on meeeee, take me oooon', 'a sabbatical year soon', 'your pet to the vet',
+'a dive in Niagara Falls', 'your last breath sometime in the next 90 years', 'a bottle of Vodka to work', 'me to visit Rome',
+'some time to see all my random messages']
+
+const caseLose = ['all hope', 'your driver\'s licence','your favorite game', 'your religion', 'your sanity', 'your car keys',
+'your dignity', 'your life savings', 'an umbrella on a rainy day', 'your sense of style', 'the tip of your eleventh finger',
+'a bet with your fiancée', 'your favorite sock', 'yourself', 'everything']
+
+const caseGive = ['a firm handshake on a snake', 'up giving up', 'that bad movie another shot', 'dancing the mambo a chance',
+'bad gifts to your mother-in-law', 'some thought about the universe\'s heat death', 'a good grade to a bad student', 'me a 5 star review in the App Store',
+'it to me baby. Aham, aham', 'a friend a hug soon', 'your dog a break', 'in to fear', 'a mouth to mouth breathing on a toothless lady', 'me some change',
+'your cat a sibling']
+
+const caseBecome = ['filthy rich', 'even poorer', 'a movie star', 'a rock star', 'a K-Pop idol', 'a father of triplets', 'a mother of triplets', '27.8 pounds heavier',
+'thinner than a Victoria\'s Secret Angel', 'a handmaid in Buckingham palace', 'friends with a squirrel', 'the next emperor of Japan', 'heir to the iron throne',
+'lost in a jungle', 'one with the Force']
+
+
+const sentence = () =>{
+    //defining a random number to be the index to be used on each array on final sencence 
+    const randomStarter = Math.floor(Math.random() * starter.length);
+    const randomVerb = Math.floor(Math.random() * verbs.length);
+    const randomBe = Math.floor(Math.random() * caseBe.length);
+    const randomHave = Math.floor(Math.random() * caseHave.length);
+    const randomMake = Math.floor(Math.random() * caseMake.length);
+    const randomTake = Math.floor(Math.random() * caseTake.length);
+    const randomLose = Math.floor(Math.random() * caseLose.length);
+    const randomGive = Math.floor(Math.random() * caseGive.length);
+    const randomBecome = Math.floor(Math.random() * caseBecome.length);
+    let sentenceArray = []; // defining an empty array to store the final sentence
+    sentenceArray.push(starter[randomStarter]); //push random starter 
+    sentenceArray.push('you will');
+    sentenceArray.push(verbs[randomVerb]); //push a random verb
```

Comment:
You can initialize elements of the array in one statement like so:
```
const sentenceArray = [
    starter[randomStarter],
    "you will",
    verbs[randomVerb],
]
```
This will help make it clear that these values belong in these spots and these spots only.

Ideally, use `.push()` when the insertion order is only known at run-time